### ISSUE TITLE
Text Provider ensure text starts with an uppercase letter

### ISSUE THREAD

**Issue by Antonimo** [fzaninotto/Faker#527]
Many Languages don't have capital/uppercase letters (like hebrew).
Also I think people that use Faker to fill random data don't really care about that.
I suggest make it optional, so we could do something like this:

```php

class HebrewTextProvider extends \Faker\Provider\Text
{
    protected $ensureTextStartsWithUppercase = false;

```

**Comment by fzaninotto** [comment]
Great idea. Would you like to work on a PR for that?

**Comment by Antonimo** [comment]
just add an if!

**Comment by Antonimo** [comment]
https://github.com/fzaninotto/Faker/pull/528

**Comment by fzaninotto** [comment]
Fixed by #778
